_posts/2018-26-04-elixir-post.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title:  "Working With Object Oriented and Functional Programming Without Losing Your Mind"
+date:   2018-04-26 12:00:00 -0400
+categories: elixir coding ruby oo functional
+author: Tania Paiva
+---
+
+At Forward Financing we work with several technologies, our stack is pretty mixed.
+Rails, Phoenix, React, Redux just to name a few - which means we have to switch from one development tool or environment to the next several times a day on a regular basis.
+
+In practice, this means you could find yourself developing a new feature in the Elixir-Phoenix environment in the morning, and improving existing Ruby on Rails code in the afternoon.
+
+The main thing with this switching is the programming paradigm that each of this technologies uses is totally
+different.
+
+### Object Oriented Programming (OOP) and Functional Programming (FP)
+
+A program has two main components, the data and the behavior.
```

Comment:
Maybe replace the `,` with a`:`, in order to direct the focus of the reader to the following elements? It would be: `A program has two main components: the data and the behavior.` Other than that all looks awesome!! Love it Tania!